articles/iot-central/core/howto-manage-users-roles-with-rest-api.md

```diff
@@ -56,6 +56,7 @@ The response to this request looks like the following example that includes the
     }
   ]
 }
+Please note that this command only shows roles that are associated with an application and not a custom organization level role.
```

Comment:
Hi Adaeze,

Can you move the text you added outside of the code snippet and use the following syntax to add the note:

```
> [!NOTE]
> This command only shows roles that are associated with an application and not a custom organization level role.
```